test/videojs-http-streaming.test.js

```diff
@@ -4503,6 +4503,56 @@ QUnit.test('configures eme for HLS on source buffer creation', function(assert)
   }, 'set source eme options');
 });
 
+QUnit.test('eme handles keystatuschange where status is not usable', function(assert) {
```

Comment:
Would be good to have one more test where status is usable.

Reply:
added.